packages/vscode-apollo/README.md

```diff
@@ -1,18 +1,154 @@
+title: Editor plugins
+description: How to get the most out of your editor with Apollo
+---
 
-## Features
+By design, GraphQL has the ability to create incredible developer experiences thank's to its strongly typed schema and query language. The Apollo GraphQL Platform brings these possibilities to life with deep editor integrations in Apollo GraphQL for VS Code. This extension brings an all-in-one tooling experience for developing apps with Apollo, including features like:
 
-- Adds syntax highlighting for GraphQL files and `gql` templates inside JavaScript files
-- Code-completes fields, arguments, types, and variables in your queries
-- Detects and loads client side schemas and validates client side field usage in operations
-- Displays performance statistics from [Apollo Engine](https://www.apollographql.com/docs/engine/) inline with your queries
-- Jump-to-definition for fragments and schema types
-- Detects fragment references and shows them next to definitions
+- Instant feedback and [intelligent autocomplete](#autocomplete) for fields, arguments, types, and variables as you write queries
+- Seamlessly manage your client side schema alongside your remote one
+- [See performance information](#performance-insights) inline with your query definitions
+- Loads GraphQL schemas and queries automatically from an Apollo Config file
+- Adds [syntax highlighting](#syntax) for GraphQL files and gql templates inside JavaScript files
+- Detects and loads client-side schemas and validates client side field usage in operations
+- [Jump-to-definition](#jump-to-def) for fragments and schema types
+- Supports [local](#local-schemas) and [client-only](#client-only-schemas) schemas
+- Allows [switching schema tags](#commands) to work on upcoming features
+- And more...
 
-## Configuration
+<img src="./images/marketplace/jump-to-def.gif" width="80%" style="margin: 5%" alt="Using jump to definition on a fragment">
 
-Please refer to the [Editor Plugin](https://www.apollographql.com/docs/platform/editor-plugins.html) docs for configuration instructions.
+<h2 id="getting-started">Getting Started</h2>
 
-## Issues
+To get started, first **install the Apollo GraphQL extension**. After installation, GraphQL syntax highlighting should automatically be enabled for `.graphql`, `.gql`, `.js` and `.ts` file types.
 
-Please report any issues you find to the [Apollo Tooling](https://github.com/apollographql/apollo-tooling) repo. Pull requests are always welcome!
+<h3 id="linking-a-schema">Linking a schema</h2>
+
+To get all of the benefits of the VS Code experience, its best to link the schema that is being developed against. The best way to do that is by [publishing a schema](./schema-registry.html#publish) to the Apollo schema registry. Once that is done, two steps are needed:
+
+1. Create an `apollo.config.js` at the root of the project
+2. Copy an API key from the Engine dashboard of the published service
+
+<h3 id="apollo-config">Setting up an Apollo config</h3>
```

Comment:
I think we should be linking to external docs for config stuff, mostly for the sake of having only one thing to maintain and keep track of.

I _do_ like having something here that scratches the surface, but I think we're better off linking them to something more complete and not having 2+ things to maintain.